examples/passport-login/src/authentication-strategies/types.ts

```diff
@@ -91,3 +97,34 @@ export const mapProfile = function (user: User): UserProfile {
   };
   return userProfile;
 };
+
+export class PassportInitMW implements Provider<Interceptor> {
```

Comment:
There is no need to wrap a simple Express middleware as provider class for local interceptors. You simply bind it to the context, or create a binding for it:
```ts
app.bind('PassportInitMW').to(toInterceptor(passport.initialize());
// or
const binding = new Binding('PassportInitMW').to(toInterceptor(passport.initialize());
```

Reply:
agreed